.github/workflows/release.yml

```
@@ -131,6 +131,7 @@ jobs:
     strategy:
       matrix:
         arch: ['x86_64', 'aarch64']
+        toolkit: ['12.9', '13.0']
```

The action needs the `cuda-` prefix to know to install cuda.

```suggestion
        toolkit: ['cuda-12.9', 'cuda-13.0']
```